INN/umbrella-inndev
# Style embed feed template

## Changes

This pull request makes the following changes:

- Adds support for typekit
- Adds basic styling for the embed feed
   - Adds support for 4 more query params
      - `horizontal=true` will set the saved links to be displayed in a horizontal column
      - `show_link_descriptions=true` will display each saved links description
      - `hide_header=true` will hide the tag header and `stories from this project` text
      - `rows=2` and `rows=3` will set the saved links to display x amount of rows per column 

@benlk Note that this is just another "base" PR so that the Amplify team can view it on staging and then provide feedback on next steps. There are still to-do's before prod, such as dequeueing unused scripts and styles (#132).

Vertical:
![Screen Shot 2019-12-13 at 1 09 27 PM](https://user-images.githubusercontent.com/18353636/70821758-e660bf00-1da9-11ea-978d-9c34717df924.png)

Horizontal:
![Screen Shot 2019-12-13 at 1 09 56 PM](https://user-images.githubusercontent.com/18353636/70821757-e660bf00-1da9-11ea-8c4b-096462901df4.png)

2 rows:
![Screen Shot 2019-12-13 at 1 10 36 PM](https://user-images.githubusercontent.com/18353636/70821793-fc6e7f80-1da9-11ea-929b-693e706f9f30.png)

3 rows:
![Screen Shot 2019-12-13 at 1 10 43 PM](https://user-images.githubusercontent.com/18353636/70821792-fc6e7f80-1da9-11ea-8dfd-31bb4be41aac.png)

## Why



For #127

## Testing/Questions

Features that this PR affects:

- Amplify feed


Questions that need to be answered before merging:

- [ ] Can you break any of the query params?
- [ ] Should we add any other query params?

Steps to test this PR:

1. View the feed template
2. Try out each query param and see if any break it
